doc/source/skips/template.rst

```diff
@@ -0,0 +1,83 @@
+.. _skip-template::
+
+==============================
+SKIP Template and Instructions
+==============================
+
+:Author: <list of authors' real names and, optionally, email addresses>
+:Status: <Draft | Active | Accepted | Deferred | Rejected | Withdrawn |
+          Final | Superseded>
+:Type: <Standards Track | Process>
+:Created: <date created on, in yyyy-mm-dd format>
+:Resolved: <date resolved, in yyyy-mm-dd format>
+:Resolution: <url> (required for Accepted | Rejected | Withdrawn)
+:Version effective: <version-number> (for accepted SKIPs)
+
+Abstract
+--------
+
+The abstract should be a short description of what the SKIP will achieve.
+
+
+Detailed description
+--------------------
+
+This section describes the need for the SKIP. It should describe the
+existing problem that it is trying to solve and why this SKIP makes the
+situation better. It should include examples of how the new functionality
+would be used and perhaps some real-world use cases.
+
+
+Implementation
+--------------
+
+This section lists the major steps required to implement the SKIP. Where
+possible, it should be noted where one step is dependent on another, and which
+steps may be optionally omitted. Where it makes sense, each step should
+include a link related pull requests as the implementation progresses.
+
+Any pull requests or developmt branches containing work on this SKIP should
+be linked to from here. (A SKIP does not need to be implemented in a single
+pull request if it makes sense to implement it in discrete phases).
+
+
+Backward compatibility
+----------------------
+
+This section describes the ways in which the SKIP breaks backward
+compatibility.
+
+
+Alternatives
+------------
+
+If there were any alternative solutions to solving the same problem, they
+should be discussed here, along with a justification for the chosen
+approach.
+
+
+Discussion
+----------
+
+This section may just be a bullet list including links to any discussions
+regarding the SKIP, but could also contain additional comments about that
+discussion:
+
+- This includes links to mailing list threads or relevant GitHub issues.
+
+
+References and Footnotes
+------------------------
+
+.. [1] Each SKIP must either be explicitly labeled as placed in the public
```

Comment:
Why give a choice here?  OK to require a single license for enhancement proposals.

Reply:
Which one? =) Should we go with CC0+BY? =)

Reply:
What does it mean to have a SKIP that is CC0?  What would people do with it?  I would almost CC0 SKIP0 and then CC-BY all the others.

Reply:
Maybe someone will take a particularly beautiful sentence I write and develop a poem from it, I dunno! :joy: I think the points of the CC0+BY post apply here: a courteous request, not a license, is the correct way to enforce attribution for documents like these.